src/toil/jobStores/aws/jobStore.py

```diff
@@ -450,10 +450,10 @@ def _readFromUrl(cls, url, writable):
             srcKey.get_contents_to_file(writable)
         finally:
             srcKey.bucket.connection.close()
-        return srcKey.size
+        return srcKey.size, False
```

Comment:
Same here.  Should we check the permissions of the file copied with `shutil.copyfileobj` or is this always guaranteed to be `False`?

Reply:
I was talking with @adamnovak and I think since AWS objects don't have executability we decided to set it always to False.